.github/workflows/scripts/govulncheck-run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euo pipefail
+
+mkdir -p ./govulncheck 2>/dev/null
+
+# Get all package directories
+ALL_PKG_DIRS=$(go list ./...)
+
+# Initialize failure flag
+FAILED=0
+
+# Repository prefix to remove from package names
+REPO_PREFIX=$(go list -m)
+
+# Run govulncheck for each package
+for pkg in $ALL_PKG_DIRS; do
+  OUTPUT_FILE="./govulncheck/$(echo "$pkg" | sed "s|^$REPO_PREFIX/||" | tr '/' '_').sarif"
+  echo -e "\nRunning govulncheck for package $pkg"
+  if ! govulncheck ${GOVULN_OPTS:-} "$pkg" > "$OUTPUT_FILE"; then
```

Comment:
Nit: Can we style this `if` with brackets to make the style in this file consistent?

Reply:
I'm not bash expert, but, I think this is the preferred style in this case, see https://google.github.io/styleguide/shellguide.html#checking-return-values.  That said that style guide has some other recommendations that the script is not following...

My understanding here is that since there is no use of conditionals the code shouldn't use square brackets. In order to use brackets here the code would have to either add $() or store the result in a variable to compare the result of the command.